pkg/docker/util.js

```diff
@@ -595,6 +595,63 @@
         return deferred.promise();
     };
 
+    util.delete_image_confirm = function confirm(client, image) {
+        var deferred = $.Deferred();
+        var image_id = image.image_id !== undefined ? image.image_id : image.Id;
+        var image_name = image.name !== undefined ? image.name :
+            util.render_container_name(image.RepoTags[0]);
+
+        $('#delete-image-confirmation-dialog-title').text(
+            cockpit.format(_("Please confirm deletion of $0"), image_name));
+        $('#delete-image-confirmation-dialog-containers>table>tbody').empty();
+        client.containers_for_image(image_id).then(function(containers) {
+            var running_containers = [];
+
+            $(containers).each(function(index, value) {
+                var container = client.containers[value.Id];
+                var $row = $('<tr />')
+                    .append($('<td />').text(container.Name.startsWith('/') ?
+                                             container.Name.substring(1) : container.Name))
+                    .append($('<td />').text(util.render_container_status(container.State)));
+                $('#delete-image-confirmation-dialog-containers>table>tbody').append($row);
```

Comment:
OK OK, too much other jQuery stuff here, so maybe let's not port it over all at once.

Reply:
Should I try to throw it out ?